api/api/signals.py

```diff
@@ -2,24 +2,32 @@
 # Created by Wazuh, Inc. <info@wazuh.com>.
 # This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
 import asyncio
+import contextlib
 import logging
 import os
 import uuid
 from functools import wraps
-from typing import AsyncGenerator, Callable
+from typing import Callable
 
-from aiohttp import web
-
-from api.constants import INSTALLATION_UID_KEY, INSTALLATION_UID_PATH, UPDATE_INFORMATION_KEY
+from connexion import ConnexionMiddleware
 from wazuh.core import common
 from wazuh.core.cluster.utils import running_in_master_node
 from wazuh.core.configuration import update_check_is_enabled
 from wazuh.core.manager import query_update_check_service
 
+from api import configuration
+from api.constants import (
+    INSTALLATION_UID_KEY,
+    INSTALLATION_UID_PATH,
+    UPDATE_INFORMATION_KEY,
+)
 
 ONE_DAY_SLEEP = 60*60*24
 
 logger = logging.getLogger('wazuh-api')
+start_stop_logger = logging.getLogger('start-stop-api')
```

Comment:
I tried using `logger` instead of `start_stop_logger` inside the lifespan_handler and it seems to work fine. Although this logger was not created in this PR, I think we can get rid of it as @GGP1 suggested.